ldap/servers/slapd/fedse.c

```diff
@@ -2950,15 +2950,10 @@ setup_internal_backends(char *configdir)
         dse_register_callback(pfedse, SLAPI_OPERATION_ADD, DSE_FLAG_PREOP, &plugins, LDAP_SCOPE_SUBTREE, "(objectclass=nsSlapdPlugin)", check_plugin_path, NULL, NULL);
 
         be = be_new_internal(pfedse, "DSE", DSE_BACKEND, &fedse_plugin);
```

Comment:
You could remove "Slapi_Backend *be" from the top of the function, and change this call to just be:  

    be_new_internal(pfedse, "DSE", DSE_BACKEND, &fedse_plugin);

I don't think that will trigger a complier warning, but best to check.  Otherwise ack from me.

For other reviewers ->  be_new_internal() adds that backend to a global list (backends in backend_manager.c), and it does get freed later.

Reply:
Seems like a good approach to me @mreynolds389 :)

Reply:
Makes sense, I should have removed that with the previous change.